build/scripts/list_containers.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# pull all external references to container images
```

Comment:
I'm not clear on the purpose of this script, could you clarify? As it stands it looks like it only does
```
yq -r '.components[].image | strings' **/devfile.yaml | sort | uniq
```

Reply:
yeah, good call. yq script pushed instead of lengthy bash thing.